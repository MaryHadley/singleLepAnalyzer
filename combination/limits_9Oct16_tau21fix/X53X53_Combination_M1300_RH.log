000 (1295372.000.000) 10/09 22:41:05 Job submitted from host: <192.168.63.249:41533>
...
001 (1295372.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.227:45329>
...
006 (1295372.000.000) 10/09 22:41:34 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2268  -  ResidentSetSize of job (KB)
...
006 (1295372.000.000) 10/09 22:46:34 Image size of job updated: 283312
	145  -  MemoryUsage of job (MB)
	147600  -  ResidentSetSize of job (KB)
...
006 (1295372.000.000) 10/09 23:31:41 Image size of job updated: 283312
	145  -  MemoryUsage of job (MB)
	147612  -  ResidentSetSize of job (KB)
...
006 (1295372.000.000) 10/10 14:23:12 Image size of job updated: 297592
	157  -  MemoryUsage of job (MB)
	160360  -  ResidentSetSize of job (KB)
...
006 (1295372.000.000) 10/10 14:28:13 Image size of job updated: 297592
	157  -  MemoryUsage of job (MB)
	160364  -  ResidentSetSize of job (KB)
...
005 (1295372.000.000) 10/10 14:32:32 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 15:49:40, Sys 0 00:00:05  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 15:49:40, Sys 0 00:00:05  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1594  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1594  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31729        1   3824812
	   Memory (MB)          :      157     3072      3072
...
