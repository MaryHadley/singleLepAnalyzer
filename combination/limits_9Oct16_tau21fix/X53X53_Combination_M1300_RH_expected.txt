# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.069776   0.034998    0.14593  0.048144    0.10244
